pkg/ansible/controller/types.go

```diff
@@ -96,13 +99,15 @@ func NewStatusFromMap(sm map[string]interface{}) Status {
 
 type ResourceStatus struct {
 	Status         `json:",inline"`
+	Phase          string   `json:"phase"`
 	FailureMessage string   `json:"reason,omitempty"`
 	History        []Status `json:"history,omitempty"`
 }
 
 func UpdateResourceStatus(sm map[string]interface{}, je eventapi.StatusJobEvent) (bool, ResourceStatus) {
 	newStatus := NewStatusFromStatusJobEvent(je)
 	oldStatus := NewStatusFromMap(sm)
+	phase := ""
```

Comment:
You could set phase = running and only change it if Failures > 0 and not have the else statement. I would suggest that we may want to do it that way.